implementation/core/src/main/java/io/smallrye/faulttolerance/core/bulkhead/FutureThreadPoolBulkhead.java

```diff
@@ -60,8 +60,8 @@ private Future<V> doApply(InvocationContext<Future<V>> ctx) throws Exception {
                 }
 
                 if (LOG.isTraceEnabled()) {
-                    LOG.trace("Cancelling bulkhead task," + (event.interruptible ? "" : " NOT")
-                            + " interrupting executing thread");
+                    LOG.tracef("Cancelling bulkhead task, %s interrupting executing thread",
```

Comment:
An extra space before `%s` :-)

```suggestion
                    LOG.tracef("Cancelling bulkhead task,%s interrupting executing thread",
```

Reply:
Ah... I see now. Fixed.